src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.InputHelper.cs

```diff
@@ -60,10 +60,9 @@ public static void Tap(UIElement element)
 					finger.Press(center);
 				});
 
-#if NETFX_CORE
-				// On Windows, We need to wait a bit before releasing for the popup to open.
-				Thread.Sleep(600);
-#endif
+				// UNO specific - we sometimes need to explicitly wait between press and release, e.g. in CalendarDatePickerIntegrationTests
+				Thread.Sleep((int)waitBetweenPressRelease);
```

Comment:
I'm surprised that this has an effect on uno, since this code may be executed on the UI thread, but maybe not in this case?

Reply:
I guess I wrote it out poorly. What I meant was that the change was needed even though most of the code is ported from WinUI, so we probably shouldn't need to change it (although the change looks super random). In fact, the waiting parameter workaround is not used in Uno, but specificially for running the tests on Windows.